windows/MinBackgroundListener.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ReactNative.Bridge;
+using System;
+
+namespace CodePush.ReactNative
+{
+    internal class MinBackgroundListener : ILifecycleEventListener
+    {
+        private DateTime? _lastSuspendDate;
+        private Action _resumeAction;
+
+        internal int MinimumBackgroundDuration { get; set; }
+
+        internal MinBackgroundListener(Action resumeAction, int minBackgroundDuration)
```

Comment:
I think that abbreviations are generally frowned upon in .NET for class names and methods, so the use of "Min" here feels a little odd.